<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrlItemInit</class>
 <widget class="QDialog" name="PrlItemInit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initialise new primitive list item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="genLabel">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>select primlist item type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="prliType"/>
     </item>
     <item>
      <widget class="QLabel" name="custLabel">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>select a custom scan</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="prliCustType"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="flavourLayout">
       <item>
        <widget class="QRadioButton" name="flavourI3Button">
         <property name="text">
          <string>FE-I2/3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="flavourI4Button">
         <property name="text">
          <string>FE-I4A/B</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer26">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="ComLabel">
         <property name="text">
          <string>Label</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Comments">
         <property name="font">
          <font/>
         </property>
         <property name="toolTip">
          <string>Label that will show up in tree-view after data uploading</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="OKButt">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font/>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer23">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>271</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancButt">
         <property name="font">
          <font/>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>OKButt</sender>
   <signal>clicked()</signal>
   <receiver>PrlItemInit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancButt</sender>
   <signal>clicked()</signal>
   <receiver>PrlItemInit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
